client/src/components/Overview/ImageGalleryThumbnail.jsx

```diff
@@ -1,20 +1,32 @@
 import React from 'react';
 
-var ImageGalleryThumbnail = function(props) {
-  if (props.selected) {
-    return (
-      <React.Fragment>
-        <div className='image-gallery-thumbnail'>
-          <img src={props.thumbnail_url} className="ig-thumbnail" />
-          <hr className='three-px-hr'/>
-        </div>
-      </React.Fragment>
+class ImageGalleryThumbnail extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    document.getElementById('igthumb-' + this.props.index).addEventListener('click', (event) => {
```

Comment:
Instead of selecting by ID, it might be better to pass down a selectImage function or something from the parent element. This way, you won't need to mount it or use a class component, considering you are using the parent's state anyway. Would also decouple this component from the parent.

Reply:
Then in that function, you could use the event that is passed in to access the id.

`e.target.id` would get it I think